beacon-chain/sync/validate_aggregate_proof.go

```diff
@@ -114,6 +120,8 @@ func (s *Service) validateAggregateAndProof(ctx context.Context, pid peer.ID, ms
 
 	msg.ValidatorData = m
 
+	aggregateAttestationVerificationGossipSummary.Observe(float64(prysmTime.Since(receivedTime).Milliseconds()))
```

Comment:
This can be quite noisy, we receive a lot of aggregates per slot. Doing a time duration measurement might be pretty noisy